AUTHORS.rst

```diff
@@ -0,0 +1,10 @@
+Authors
+=======
+
+Billinge Group and community contibutors.
```

Comment:
Please look in the authors of the original and add all the names that are there on the line above, and then this line